quickstarts/gradle/spring-boot-crd/build.gradle

```diff
@@ -31,7 +31,7 @@ repositories {
 dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-actuator'
     implementation 'org.springframework.boot:spring-boot-starter-web'
-    implementation 'io.fabric8:kubernetes-client:5.7.0'
```

Comment:
I'm not sure that this is going to work well. Especially considering the quickstarts.sh gradleVersion script function.

Thinking this better the awesome way to deal with this would be to actually extract the Kubernetes Client transitive dependency version and then fix the quickstarts.sh script to take care of bumping that one too. Then both, the Maven and Gradle spring-crd quickstart could benefit from this.

Reply:
You closed the issue,

> Thinking this better the awesome way to deal with this would be to actually extract the Kubernetes Client transitive dependency version and then fix the quickstarts.sh script to take care of bumping that one too. Then both, the Maven and Gradle spring-crd quickstart could benefit from this.

Can this be worked on then ?

Reply:
This should be done in scope of https://github.com/eclipse/jkube/issues/2123

Reply:
Then don't close the PR

Reply:
Have you check if by running `./scripts/quickstarts.sh version` the target plugin version changes?

Reply:
I checked and it seems to update the plugin version 
```diff
 plugins {
     id 'org.springframework.boot' version '2.5.4'
     id 'io.spring.dependency-management' version '1.0.11.RELEASE'
-    id 'org.eclipse.jkube.kubernetes' version '1.13.1'
-    id 'org.eclipse.jkube.openshift' version '1.13.1'
+    id 'org.eclipse.jkube.kubernetes' version '1.13.2'
+    id 'org.eclipse.jkube.openshift' version '1.13.2'
     id 'java'
 }
 
@@ -31,7 +31,7 @@ repositories {
 dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-actuator'
     implementation 'org.springframework.boot:spring-boot-starter-web'
-    implementation 'org.eclipse.jkube.kubernetes:org.eclipse.jkube.kubernetes.gradle.plugin:1.13.1'
+    implementation 'org.eclipse.jkube.kubernetes:org.eclipse.jkube.kubernetes.gradle.plugin:1.13.2'
 }

```

Reply:
OK; reopen it and we'll merge, this is better than nothing.

But keep in mind the other suggested approach

Reply:
I've added a comment in #2123